freestyle-usage-snippet-finder.js

```diff
@@ -117,7 +117,8 @@ SnippetFinder.prototype.write = function (readTree, destDir) {
       var commentSnippets = extractCommentSnippets(fs.readFileSync(filename, 'utf-8'));
       var snippets = naiveMerge(componentSnippets, commentSnippets);
       for (var name in snippets){
-        fs.writeFileSync(path.join(destDir, name)+path.extname(filename),
+        var windowsFriendlyName = name.replace(":notes", "_notes").replace(":usage", "_usage"); // replace : in order to have windows friendly filenames
```

Comment:
I would rather change all initial usage of `:` to a safe delimiter rather than replace it here. That way folks won't be confused by seeing `:notes` in some places but `_notes` in other places.